std/digest/crc.d

```diff
@@ -355,6 +502,42 @@ ubyte[4] crc32Of(T...)(T data)
     return digest!(CRC32, T)(data);
 }
 
+/**
+ * This is a convenience alias for $(REF digest, std,digest,digest) using the
+ * CRC64-ECMA implementation.
+ *
+ * Params:
+ *      data = $(D InputRange) of $(D ElementType) implicitly convertible to
+ *             $(D ubyte), $(D ubyte[]) or $(D ubyte[num]) or one or more arrays
+ *             of any type.
+ *
+ * Returns:
+ *      CRC64-ECMA of data
+ */
+//simple alias doesn't work here, hope this gets inlined...
+ubyte[8] crc64ECMAOf(T...)(T data)
```

Comment:
I'm not sure what I'm expected to do here. I looked at crc32Of and there is no example. Is there a piece of code I can look at to know what to do ?